### PR TITLE
Small improvements to the installation page.

### PR DESCRIPTION
**Pull request by axelsimon**
Fixes a link or two, makes capitalisation consistent, that sort of stuff.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -148,7 +148,7 @@ Python-based prerequisites
     The following outlines installing Keylime under the Python 2 environment,
     work is underway to port Keylime to Python 3.
 
-The following python packages are required:
+The following Python packages are required:
 
 * pycryptodomex>=3.4.1
```
Comment (lukehinds) [inline docs/installation.rst:153]: Change looks good , just noticed we need to update these dependencies (we dropped pycryptodome for python-cryptography) - you can fix here if you like or we can do a follow up PR.

**Review by lukehinds — APPROVED** [review]

File: docs/installation.rst (hunk 1 above)
Comment (axelsimon) [reply docs/installation.rst:153]: Let's do it here. If you give the min. version for python-cryptography i will update my PR.

Comment (axelsimon) [reply docs/installation.rst]: Just saw we still had this hanging while doing some tidying up. Let's try and close it! @lukehinds @mpeters do you now the min version for `python-cryptography`?

**Comment by axelsimon** [comment]
This PR now also fixes the [user guide's index page](https://github.com/keylime/keylime-docs/blob/master/docs/user_guide.rst) which currently ponits to the wrong URL/document name for the secure payload documentation.

File: docs/installation.rst (hunk 1 above)
Comment (mpeters) [reply docs/installation.rst]: This was now bumped to v3.3.2 for some security fixes and dependency problems.

**Pull request merged**